crates/cargo-contract/Cargo.toml

```
@@ -36,16 +36,17 @@ rust_decimal = "1.28"
 
 # dependencies for extrinsics (deploying and calling a contract)
 async-std = { version = "1.12.0", features = ["attributes", "tokio1"] }
-sp-core = "7.0.0"
-sp-weights = "4.0.0"
-pallet-contracts-primitives = "7.0.0"
-subxt = "0.25.0"
+sp-core = "11.0.0"
+sp-runtime = "12.0.0"
+sp-weights = "8.0.0"
+pallet-contracts-primitives = "12.0.0"
+subxt = { version = "0.26.0", features = ["substrate-compat"] }
```

According to the release notes this feature is on by default 
```suggestion
subxt = "0.26.0"
```